src/pocketmine/network/rcon/RCONInstance.php

```diff
@@ -77,13 +77,13 @@ class RCONInstance extends Thread{
 	 * @param resource             $socket
 	 * @param resource             $ipcSocket
 	 */
-	public function __construct($socket, string $password, int $maxClients = 50, \ThreadedLogger $logger, $ipcSocket, ?SleeperNotifier $notifier){
+	public function __construct($socket, string $password, ?int $maxClients, \ThreadedLogger $logger, $ipcSocket, ?SleeperNotifier $notifier){
```

Comment:
why nullable? it had to be specified already anyway, it should just be required

Reply:
this is the best way to fix this without causing a BCBreak. If you don't mind changing the order of the arguments, it would be better to move $maxClient backwards

Reply:
It's not causing a BC break any way you do it. The default value was already useless.

Reply:
okay